### DISCUSSION TITLE
react 從 App.jsx import 檔案中的函式，網頁卻跑不動

### DISCUSSION THREAD

**Discussion by rdcvgt** [Lidemy/forum#291] [其他問題]
最近在練習 react 不斷出現很多狀況，這次是在 react router 階段出現問題，
我原本以爲是 Router 導致連線不正常，但把問題縮小後發現是引入檔案造成網頁跑不動

從別資料夾中 import 檔案中的函式作爲 component 使用，整個網頁就會跑不動，但也沒有出現 error 的訊息
如果只是單純在 App() return 字串，或是在 App.jsx 中直接寫 component 都很正常，想請老師幫忙看一下是哪裡出了問題

附上目前 codesandbox 連結
https://codesandbox.io/s/vigilant-lake-uyy93w?file=/src/components/Header/Header.jsx

**Comment by aszx87410** [comment]
codesandbox 本身好像出了點問題，怪怪的

除此之外，你的 header component 是寫：

``` js
export default function Header() {
  return <Header>Header</Header>;
}
```

你 Header 裡面又 render Header，就無窮迴圈了，你應該是想寫 `<header>Header</header>` ?

**Reply by rdcvgt** [reply]
原來如此，我誤把 return 的 component 取到一樣的名稱導致無窮迴圈，感謝老師解答！
